src/components/JarCard/transformTransactionTime.js

```diff
@@ -0,0 +1,18 @@
+export default function transformTransactionTime(transactionDate) {
+  if (!transactionDate) return 'No donations yet';
+  const lastTransaction = Math.floor(Date.now() - new Date(transactionDate));
+  const MINUTE = 1000 * 60;
+  const HOUR = MINUTE * 60;
+  const DAY = HOUR * 24;
+  const WEEK = DAY * 7;
+  const response = 'Last donation';
+  if (lastTransaction < MINUTE) {
+    return response + ` less than a minute ago`;
+  } else if (lastTransaction < HOUR) {
+    return response + ` ${Math.floor(lastTransaction / MINUTE)} min. ago`;
+  } else if (lastTransaction < DAY) {
+    return response + ` ${Math.floor(lastTransaction / HOUR)} hrs. ago`;
+  } else if (lastTransaction < WEEK) {
+    return response + ` ${Math.floor(lastTransaction / DAY)} d. ago`;
+  } else return response + ' more than a week ago';
+}
```

Comment:
will it be reused anywhere else? Also I would suggest to move it to the utils folder

Reply:
Currently used only in one file, created new file to shorten the code. 
Also it's in `components>jarCard ` folder and there is already one function that chooses what jar image to show. Would you like me to move both of them to utils?

Reply:
I think it will be much better to move them. Also you can create utils folder in components/jarCard :)

Reply:
Fixed and noted :)